src/models/oracle/oracleEndpoint.js

```diff
@@ -126,6 +126,26 @@ const getAllOracleEndpoint = async () => {
   }
 }
 
+const checkActiveOracleEndpoint = async (oracleEndpointModel, partyIdTypeId, endpointTypeId) => {
```

Comment:
I would consider renaming this to `getAllOracleEndpointsByMatchCondition`.

The "check" infers that some logic will occur within the function, where this is actually just returning a list of endPoints that are active matching the condition.

Let me know what you think, or if you have another suggestion?

Reply:
done.